app/src/main/java/com/github/sdp/mediato/data/LocationDatabase.java

```diff
@@ -0,0 +1,99 @@
+package com.github.sdp.mediato.data;
+
+import android.util.Log;
+
+import com.github.sdp.mediato.errorCheck.Preconditions;
+import com.github.sdp.mediato.model.Location;
+import com.github.sdp.mediato.model.User;
+import com.google.firebase.database.FirebaseDatabase;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+
+public class LocationDatabase {

```

Comment:
maybe add a javadoc to the class declaration

Reply:
Addressed in latest commit (even added it for the other database classes ;)